athena-neptune/athena-neptune.yaml

```diff
@@ -90,9 +93,15 @@ Resources:
                 - glue:GetPartition
                 - glue:GetDatabase              
                 - athena:GetQueryExecution
-                - neptune-db:*
               Effect: Allow
               Resource: '*'
+            - Action:
+                - neptune-db:*
+              Effect: Allow
+              #Resource: '*'
+              Resource: !Sub
+                - arn:aws:neptune-db:${AWS::Region}:${AWS::AccountId}:${NCR}/*
+                - { NCR: !Ref NeptuneClusterResourceID }
```

Comment:
I think you are also missing S3 list bucket permissions. This was added as a requirement in new SDK versions so that we can verify that the customer owns the spill bucket (its a security protection to avoid someone sending data to a bucket they dont own due to bucket sniping)